code/player/Player.cs

```diff
@@ -242,14 +255,17 @@ public override void TakeDamage(DamageInfo info)
                 info.Damage *= 2.0f;
             }
 
-            if (info.Attacker is TTTPlayer attacker && attacker != this)
+            using (Prediction.Off())
```

Comment:
We do not want to turn off prediction here I'm pretty sure. If the FP guys didn't, we shouldn't.

https://wiki.facepunch.com/sbox/Prediction